src/containers/List/ListFilters/ListFilters.tsx

```diff
@@ -269,6 +272,13 @@ const ListFilters: React.FC<{
             return null
         }
       })}
+      {totalCount ? (
+        <p className="result-count">
+          {totalCount === 1
+            ? strings.APPLICATIONS_LIST_TOTAL_RESULTS_SINGLE
+            : strings.APPLICATIONS_LIST_TOTAL_RESULTS.replace('%1', String(totalCount))}
+        </p>
```

Comment:
This is a super-clunky way to handle pluralisation. Have made an issue to do it properly: https://github.com/openmsupply/conforma-web-app/issues/1488